# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/arcticfox/hotfix-sparc64.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- Arctic-Fox-46.3/ipc/chromium/src/build/build_config.h.vanilla	2025-10-07 18:33:45.257181736 +0200
+++ Arctic-Fox-46.3/ipc/chromium/src/build/build_config.h	2025-10-07 18:33:53.357324150 +0200
@@ -83,7 +83,7 @@
 #elif defined(__ppc__) || defined(__powerpc__)
 #define ARCH_CPU_PPC 1
 #define ARCH_CPU_32_BITS 1
-#elif defined(__sparc__) && defined(__arch64__)
+#elif (defined(__sparc__) || defined(__sparc_v9__)) && defined(__arch64__)
 #define ARCH_CPU_SPARC 1
 #define ARCH_CPU_64_BITS 1
 #elif defined(__sparc__)
